### DISCUSSION TITLE
Simple way to handle route submission in a Remix app | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#76] [Announcements]
# Simple way to handle route submission in a Remix app | Leo's dev blog – stories, insights, and ideas

Create a custom hook to manage in-route form submissions in a Remix app.

https://www.leohuynh.dev/snippets/remixing-remix-submission

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
